09-redux/classworks/Gavruk_Alexander/src/pages/Routes.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Switch, Route, Redirect } from 'react-router-dom';
+
+import useRedditApi from 'hooks/useRedditApi';
+
+import Login from 'pages/login';
+import Callback from 'pages/callback';
+import Home from 'pages/home';
+
+function Routes() {
+  const [,,isLoggedIn] = useRedditApi();
+
+  if (isLoggedIn) { 
+    return (
+      <Switch>
+        <Route path="/" exact component={() => <Home />} />
```

Comment:
as we don't pass anything anymore we can make just `component={Home}`. but let's leave it as it is here